.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
 
```

Comment:
How about adding these?

```
Style/SymbolArray:
  Enabled: false

Style/WordArray:
  Enabled: false

Layout/MultilineOperationIndentation:
  EnforcedStyle: 'indented'

Layout/MultilineMethodCallIndentation:
  EnforcedStyle: 'indented'
```

Reply:
I added the latter two. The first two are in the todo list in our main repo, so I assume that means we'd like to enable it later.